backend/JavaWebService/src/main/java/com/kainos/ea/resources/JobSpecification.java

```diff
@@ -0,0 +1,81 @@
+package com.kainos.ea.resources;
+
+public class JobSpecification {
```

Comment:
This page is a duplication of Role class - you should modify Role to include the link